bin/bst.ts

```diff
@@ -6,8 +6,9 @@ import {LoggingHelper} from "../lib/core/logging-helper";
 let Logger = "BST";
 
 Global.initializeCLI();
-LoggingHelper.info(Logger, "BST: v" + Global.version() + "  Node: " + process.version);
-LoggingHelper.info(Logger, "");
+console.log("BST: v" + Global.version() + "  Node: " + process.version);
```

Comment:
Have we considered removing these entirely?  When running the `speak` command, it might pollute the logs on  CI.